content/guides/ecommerce-connection/step0-who-is-your-connector-for.md

```diff
@@ -0,0 +1,53 @@
+# Who is your connector for?
+
+## Personnas
+
+You will find below user archetypes we use to help you guide your decisions about product features, interactions, and even visual design of your connector. By designing for the archetype — whose goals and behavior patterns are well understood — you can satisfy the broader group of people represented by that archetype.
+
+|![Julia](../../img/personas/Julia.png)|![Peter](../../img/personas/Peter.png)|![Filips](../../img/personas/Filips.png)|
```

Comment:
Maybe you should just create an image with the three personas and their name below (inside the image).
It would be better than this weird table. ;)

Reply:
Sure